packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/sounds/sounds.controller.js

```diff
@@ -67,5 +67,19 @@ export default /* @ngInject */ function TelecomTelephonyAliasConfigurationOvhPab
       });
   };
 
+  self.getFeatureType = function $onInit() {
+    if (self.number.feature?.featureType) {
+      switch (self.number.feature.featureType) {
+        case 'cloudIvr':
+          return 'SVI';
+        case 'contactCenterSolutionExpert':
+          return 'CCS';
+        default:
+          return null;
+      }
+    }
+    return null;
```

Comment:
i think we can have something like:
```suggestion
    const features = {'cloudIvr': 'SVI', 'contactCenterSolutionExpert': 'CCS'}
    return features[self.number.feature.featureType] || null;
```